### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/learn/index.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/learn/index.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/learn/index.md)

Discussion: #9193

### CODE REVIEW

**Comment by hasansezertasan** [comment]
@alperiox, @OzgunCaglarArslan @SametEmin, @mertssmnoglu, can you please leave a review?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7f597fa13decf128540d4e930179f725f59eff67 at: https://6cc6f238.fastapitiangolo.pages.dev

File: docs/tr/docs/learn/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Öğren
+
+**FastAPI** öğrenmek için giriş bölümleri ve öğreticiler burada yer alıyor.
+
+Burayı, bir **kitap**, bir kurs **kurs**, ve FastAPI öğrenmenin **resmi** ve önerilen yolu olarak düşünülebilirsiniz. 😎
```
Comment (hasansezertasan) [inline docs/tr/docs/learn/index.md:5]:
```suggestion
Burayı, bir **kitap**, bir **kurs**, ve FastAPI öğrenmenin **resmi** ve önerilen yolu olarak düşünülebilirsiniz. 😎
```

**Review by OzgunCaglarArslan — APPROVED** [review]

**Review by OzgunCaglarArslan — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 560cdd0c4cb5e68187f6bcd10b3bae847bbdcc49 at: https://72ea4a33.fastapitiangolo.pages.dev

**Review by mertssmnoglu — APPROVED** [review]
Seems like a good beginning.

**Pull request merged**